src/components/mgt-person/mgt-person.ts

```diff
@@ -6,15 +6,19 @@
  */
 
 import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+import { Presence } from '@microsoft/microsoft-graph-types-beta';
 import { customElement, html, property, query, TemplateResult } from 'lit-element';
 import { classMap } from 'lit-html/directives/class-map';
+import { BetaGraph } from '../../BetaGraph';
```

Comment:
Same here for `BetaGraph` move to graph.presnce.ts and avoid instantiating a 'BetaGraph` here